### ISSUE TITLE
How to get the internal value of enum instead of key?

### ISSUE THREAD

**Issue by AaronJan** [nestjs/graphql#227]
I'm trying to reproduce [this example](https://github.com/19majkel94/type-graphql/tree/master/examples/enums-and-unions), after 3 hours testing, I have no idea how to get the values of enum using `nestjs/graphql` (`type-graphql` works fine).

<pre><code>
## I'm submitting a...
[ ] Regression 
[ ] Bug report
[ ] Feature request
[*] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Let's say we have an enum:

```typescript
import { registerEnumType } from "type-graphql";

export enum Difficulty {
  Beginner = 1,
  Easy = 2,
  Medium = 3,
  Hard = 4,
  MasterChef = 5,
}

registerEnumType(Difficulty, {
  name: "Difficulty",
});
```

And the `resolver`:

```typescript
import {
  Resolver,
  Args,
} from '@nestjs/graphql';
import {
  Recipe,
} from './recipe.entity';
import { Difficulty } from './recipe.constant';

@Resolver(Recipe)
export class RecipeResolver {
  @Query(returns => [Recipe])
  recipes(@Args({ name: 'difficulty', type: () => Difficulty }) difficulty: Difficulty) {
    console.log(difficulty); // I'm getting the string key name
  }
}

```

Query:

```graphql
query {
  recipes (difficulty: Beginner) {
    name
  }
}
```

The console output is `"Beginner"`.

## Expected behavior

Should be `1`.

## Minimal reproduction of the problem with instructions

If it's necessary, I'll creating one.

## What is the motivation / use case for changing the behavior?

Common usage.

## Environment

<pre><code>
Nest version: 6.1.1
 
For Tooling issues:
- Node version: 10.15.3
- Platform:  Windows

Others:
- TypeScript: 3.4.3

**Comment by marvinroger** [comment]
I've been facing that exact same issue, too. Directly using `type-graphql` works fine.

**Comment by kamilmysliwiec** [comment]
It should be fixed in the @next version (`$ npm i @nestjs/graphql@next`). You will also have to update `@nestjs/core` to 6.2.0 (`$ npm i @nestjs/core@6.2.0`)

**Comment by marvinroger** [comment]
I can confirm it's fixed in `next`, thanks @kamilmysliwiec!

**Comment by kamilmysliwiec** [comment]
Published as `6.2.0` (`$ npm i @nestjs/graphql@latest`). You will also have to update `@nestjs/core` to 6.2.0 (`$ npm i @nestjs/core@latest`)

**Comment by nigelvon** [comment]
Could GraphQLDefinitionsFactory export definitions with custom enum values?
